src/main/java/net/glowstone/entity/passive/GlowVillager.java

```diff
@@ -3,73 +3,33 @@
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkNotNull;
 
-import io.netty.buffer.ByteBuf;
-import io.netty.buffer.Unpooled;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Random;
 import java.util.concurrent.ThreadLocalRandom;
-import lombok.Getter;
-import lombok.Setter;
-import net.glowstone.GlowServer;
 import net.glowstone.entity.GlowAgeable;
 import net.glowstone.entity.GlowHumanEntity;
-import net.glowstone.entity.GlowPlayer;
 import net.glowstone.entity.meta.MetadataIndex;
-import net.glowstone.inventory.GlowMerchantInventory;
-import net.glowstone.net.GlowBufUtils;
-import net.glowstone.net.message.play.game.PluginMessage;
-import net.glowstone.net.message.play.player.InteractEntityMessage;
-import net.glowstone.util.InventoryUtil;
 import org.bukkit.Location;
-import org.bukkit.Material;
 import org.bukkit.Sound;
 import org.bukkit.entity.Entity;
 import org.bukkit.entity.EntityType;
 import org.bukkit.entity.Villager;
 import org.bukkit.entity.Witch;
 import org.bukkit.event.entity.EntityDamageEvent.DamageCause;
 import org.bukkit.inventory.Inventory;
-import org.bukkit.inventory.InventoryView;
-import org.bukkit.inventory.ItemStack;
 import org.bukkit.inventory.MerchantRecipe;
 
 public class GlowVillager extends GlowAgeable implements Villager {
 
     private static final Profession[] PROFESSIONS = Profession.values();
-    private static final MerchantRecipe DEFAULT_RECIPE
-            = new MerchantRecipe(new ItemStack(Material.DIRT), 10);
-    @Getter
+
     private Career career;
-    @Getter
-    @Setter
```

Comment:
Why is this being de-Lombokified?

Reply:
Hmm I guess I reverted too much of the file, it was supposed to get rid of the merchant-interface stuff.